demos.js

```diff
@@ -446,7 +437,7 @@ const allDemos = [
         "group": "Mercury",
         "youtube": "I8pRJ0q8OlY",
         "date": "2011-12-15",
-        "screenshot": "https://content.pouet.net/files/screenshots/00058/00058262.jpg",
+        "screenshot": "https://media.demozoo.org/screens/o/77/02/1aec.7231.jpg",
```

Comment:
I would suggest either of:
https://media.demozoo.org/screens/o/58/dc/c546.7236.jpg
https://media.demozoo.org/screens/o/05/07/4df5.7241.jpg
https://media.demozoo.org/screens/o/5b/66/77da.7238.jpg